.github/workflows/build-image.yaml

```diff
@@ -9,7 +9,7 @@ on:
   pull_request:
 concurrency:
   group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
-  cancel-in-progress: true
+  cancel-in-progress: ${{ !contains(github.ref, 'main/')}}
```

Comment:
The same should be added for v* in the case where we do releases in quick succession.